.storybook/3.COLORS.stories.mdx

```diff
@@ -0,0 +1,182 @@
+  var(--color-error-inverse);
+  var(--color-error-disabled);
+  var(--color-warning-default);
+  var(--color-warning-alternative);
+  var(--color-warning-muted);
+  var(--color-warning-inverse);
+  var(--color-warning-disabled);
+  var(--color-success-default);
+  var(--color-success-alternative);
+  var(--color-success-muted);
+  var(--color-success-inverse);
+  var(--color-success-disabled);
+  var(--color-info-default);
+  var(--color-info-alternative);
+  var(--color-info-muted);
+  var(--color-info-inverse);
+  var(--color-info-disabled);
+}
+```
+
+### **Component colors** (tier 3)
+
+Another level of abstraction is component tier colors that you can define at the top of your styles and use at the component specific level.
+
+```scss
+.button {
+  --color-background-primary: var(--color-primary-default);
+  --color-text-primary: var(--color-primary-inverse);
+  --color-border-primary: var(--color-primary-default);
+
+  --color-background-primary-hover: var(--color-primary-alternative);
+  --color-border-primary-hover: var(--color-primary-alternative);
+
+  .btn-primary {
+    background-color: var(--color-background-primary);
+    color: var(--color-text-primary);
+    border: 1px solid var(--color-border-primary);
+
+    &:hover {
+      background-color: var(--color-background-primary-hover);
+      border: 1px solid var(--color-border-primary-hover);
+    }
+
+    /** btn-primary css continued... */
+  }
+}
+```
+
+## Takeaways
+
+- Do not use static HEX values in your code. Use the [Theme Colors](#theme-colors-tier-2). If one does not exist for your use case ask the designer or [create an issue](https://github.com/MetaMask/metamask-extension/issues/new) and tag it with a `design-system` label.
+- Make sure the design token you are using is for it's intended purpose. Please refer to the description of each token in [`@metamask/design-tokens`](https://github.com/MetaMask/design-tokens/blob/main/src/figma/tokens.json#L329-L554).
+
+### ❌ Don't do this
+
+Don't use static hex values or brand color tokens in your code.
+
+```css
+/**
+* Don't do this
+* Static hex values create inconsistency and will break UI when using dark mode
+**/
+.card {
+  background-color: #ffffff;
+  color: #24272a;
+}
+
+/**
+* Don't do this
+* Not theme compatible and will break UI when using dark theme
+**/
+.card {
+  background-color: var(--brand-colors-white-white000);
+  color: var(--brand-colors-grey-grey800);
+}
+```
+
+### ✅ Do this
+
+Do use component tiered and [Theme Colors](#theme-colors-tier-2) in your styles and code
+
+```css
+.card {
+  --color-background: var(--color-background-default);
+  --color-text: var(--color-text-default);
+
+  background-color: var(--color-background);
+  color: var(--color-text);
+}
+```
+
+<br />
+
+## References
+
+- [`@metamask/design-tokens`](https://github.com/MetaMask/design-tokens)
+- [Figma Brand Colors Library](https://www.figma.com/file/cBAUPFMnbv6tHR1J8KvBI2/Brand-Colors?node-id=0%3A1)(internal use only)
```

Comment:
NIT:
```suggestion
- [Figma Brand Colors Library](https://www.figma.com/file/cBAUPFMnbv6tHR1J8KvBI2/Brand-Colors?node-id=0%3A1) (internal use only)
```
could add spaces between link and rounded brackets

Reply:
Done!

Reply:
Updated in https://github.com/MetaMask/metamask-extension/pull/13657/commits/bfbb6d6cdf98c91129ae2c256ce44383365357b8